DatadogSessionReplay/Tests/Feature/RUMContextReceiverTests.swift

```diff
@@ -115,18 +95,69 @@ class RUMContextReceiverTests: XCTestCase {
         XCTAssertEqual(rumContexts[1].viewServerTimeOffset, 345)
     }
 
-    func testWhenMessageDoesntContainRUMBaggage_itCallsFallback() {
-        let context = DatadogContext.mockAny()
-        let message = FeatureMessage.context(context)
```

Comment:
blocking/ With this removal, we're no longer covering the behaviour of `RUMContextReceiver` acknowledging the "context message" - I mean [this line](https://github.com/DataDog/dd-sdk-ios/blob/101fe09b2dd1d444e7cca262874a5ffa3396ea40/DatadogSessionReplay/Sources/Feature/RUMContextReceiver.swift#L60).

These tests seem to suffer the same problem as `WebViewLogReceiverTests` - what we test is integration with mocked core, instead of testing the standalone receiver.